.gitignore

```diff
@@ -13,7 +13,6 @@ __diff_output__
 /docs/export
 /test/regressions/screenshots
 build
```

Comment:
I assume we do no longer need it.

```suggestion
```

Reply:
I assume you meant to remove `dist`, right? I didn't remove it because I was thinking that some team members could still have this directory from previous releases, so they would have to manually delete it.

Reply:
Alright, sounds great